### ISSUE TITLE
[BUG][DISTRIBUTION][2.0.0]Still seeing requirements of admin:admin when curling api/status after reverting #943

### ISSUE THREAD

**Issue by peterzhuamazon** [opensearch-project/security-dashboards-plugin#952]
```
[2022-04-18T19:09:39,832][INFO ][o.o.c.c.ClusterBootstrapService] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] no discovery configuration found, will perform best-effort cluster bootstrapping after [3s] unless existing cluster-manager is discovered
Exception in thread "Attach Listener" [2022-04-18T19:09:39,913][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] java.lang.reflect.InvocationTargetException
[2022-04-18T19:09:39,914][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2022-04-18T19:09:39,914][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2022-04-18T19:09:39,914][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2022-04-18T19:09:39,915][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2022-04-18T19:09:39,915][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndStartAgent(InstrumentationImpl.java:513)
[2022-04-18T19:09:39,915][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.instrument/sun.instrument.InstrumentationImpl.loadClassAndCallAgentmain(InstrumentationImpl.java:535)
[2022-04-18T19:09:39,915][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] Caused by: java.security.AccessControlException: access denied ("java.lang.RuntimePermission" "accessClassInPackage.jdk.internal.org.objectweb.asm")
[2022-04-18T19:09:39,916][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.base/java.security.AccessControlContext.checkPermission(AccessControlContext.java:472)
[2022-04-18T19:09:39,916][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.base/java.security.AccessController.checkPermission(AccessController.java:897)
[2022-04-18T19:09:39,916][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.base/java.lang.SecurityManager.checkPermission(SecurityManager.java:322)
[2022-04-18T19:09:39,916][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.base/java.lang.SecurityManager.checkPackageAccess(SecurityManager.java:1238)
[2022-04-18T19:09:39,916][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:174)
[2022-04-18T19:09:39,917][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
[2022-04-18T19:09:39,917][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at Log4jHotPatch.asmVersion(Log4jHotPatch.java:71)
[2022-04-18T19:09:39,917][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	at Log4jHotPatch.agentmain(Log4jHotPatch.java:93)
[2022-04-18T19:09:39,917][WARN ][stderr                   ] [dev-dsk-zhujiaxi-2a-5662414e.us-west-2.amazon.com] 	... 6 more
```

After [reverting](https://github.com/opensearch-project/security-dashboards-plugin/pull/950) the Security Change last week, we are still seeing requirements of admin:admin when curling api/status.

```
% curl localhost:5601/api/status
{"statusCode":401,"error":"Unauthorized","message":"Authentication required"}%
```

**Comment by peterzhuamazon** [comment]
OS: https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.0.0-rc1/2525/linux/x64/tar/dist/opensearch/opensearch-2.0.0-rc1-linux-x64.tar.gz
OSD: https://ci.opensearch.org/ci/dbc/distribution-build-opensearch-dashboards/2.0.0-rc1/2962/linux/x64/tar/dist/opensearch-dashboards/opensearch-dashboards-2.0.0-rc1-linux-x64.tar.gz

**Comment by cliu123** [comment]
@peterzhuamazon I got the expected behavior at the HEAD of main branch of security-dashboards-plugin:
```
% curl http://localhost:5601/api/status
{"name":"osd","uuid":"4446f809-4c9e-4934-aed4-75e880265324","version":{"number":"2.0.0","build_hash":"920cd5e45d28784ebd6406e332d2275efecd6026","build_number":1901,"build_snapshot":false},"status":{"overall":{"since":"2022-04-18T20:30:41.843Z","state":"green","title":"Green","nickname":"Looking good","icon":"success","uiColor":"secondary"},"statuses":[{"id":"core:opensearch@2.0.0","message":"OpenSearch is available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"core:savedObjects@2.0.0","message":"SavedObjects service has completed migrations and is available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:advancedSettings@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:bfetch@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:charts@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:embeddable@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:discover@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:devTools@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:expressions@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:indexPatternManagement@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:inspector@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:legacyExport@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:management@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:opensearchDashboardsOverview@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:opensearchDashboardsReact@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:opensearchDashboardsUtils@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:opensearchUiShared@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:opensearchDashboardsUsageCollection@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:share@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:savedObjects@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:navigation@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:savedObjectsManagement@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:uiActions@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:urlForwarding@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visDefaultEditor@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visualizations@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visualize@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:apmOss@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:console@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:data@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:dashboard@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:home@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:inputControlVis@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:opensearchDashboardsLegacy@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:regionMap@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:mapsLegacy@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:timeline@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:usageCollection@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeMarkdown@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeMetric@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeTable@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeTagcloud@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeTimeline@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeVega@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeVislib@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:visTypeTimeseries@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:securityDashboards@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"},{"id":"plugin:tileMap@2.0.0","message":"All dependencies are available","since":"2022-04-18T20:30:41.843Z","state":"green","icon":"success","uiColor":"secondary"}]},"metrics":{"last_updated":"2022-04-18T20:30:39.836Z","collection_interval_in_millis":5000,"os":{"platform":"linux","platformRelease":"linux-5.10.25-linuxkit","load":{"1m":0.98,"5m":0.3,"15m":0.1},"memory":{"total_in_bytes":15733710848,"free_in_bytes":10580213760,"used_in_bytes":5153497088},"uptime_in_millis":1441425990,"distro":"Debian","distroRelease":"Debian-10.9"},"process":{"memory":{"heap":{"total_in_bytes":98353152,"used_in_bytes":69825344,"size_limit":2197815296},"resident_set_size_in_bytes":152227840},"pid":26,"event_loop_delay":0.3662998676300049,"uptime_in_millis":29550.9785},"response_times":{"avg_in_millis":0,"max_in_millis":0},"concurrent_connections":4,"requests":{"disconnects":0,"total":0,"statusCodes":{},"status_codes":{}}}}%
